test/setup/versions.tf

```
@@ -31,17 +31,17 @@ terraform {
 
     null = {
       source  = "hashicorp/null"
-      version = "~> 2.1"
+      version = "3.2.0"
```

Avoid needing to bump in the future

```suggestion
      version = ">= 2.1"
```